package.json

```diff
@@ -58,9 +58,10 @@
     "mkdirp": "^0.5.1",
     "mockery": "^2.1.0",
     "preact": "^10.1.1",
-    "preact-i18n": "^1.4.1",
+    "preact-context-provider": "^2.0.0-preactx.2",
+    "preact-i18n": "^2.0.0-preactx.2",
     "preact-render-to-string": "^5.1.3",
-    "preact-router": "^3.1.0",
+    "preact-router": "github:zimbra/preact-router#3.1.0_base_path_support",
```

Comment:
What's up with these additions to package.json? I don't see where you're using them.

Reply:
They are shimmed modules in `src/shims/index.js` for Zimlets to use.